modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationRegistry.java

```diff
@@ -299,6 +314,15 @@ public Void visitInnerNode(String key, InnerNode newRoot) {
         return CompletableFuture.allOf(resultFutures);
     }
 
+    /**
+     * Notifies all listeners of the current configuration.
+     *
+     * @return Future that must signify when processing is completed.
+     */
+    public CompletableFuture<Void> notifyCurrentConfigurationListeners() {
```

Comment:
I think a better comment is required. That this method treats all old values as nulls and exists for node start routine exclusively.

Reply:
Hope it's better now

Reply:
Description is somewhat cryptic I think, but it's fine.